### PR TITLE
added background and eye functionality

### PR DESCRIPTION
**Pull request by anmol-284**
## 📋 Description

Added a background to the main page for improved visual appeal. Integrated eye functionality in the form for toggling password visibility.

## 🔨 Changes Made

Added a new background to the main page to enhance the overall look.
Implemented eye functionality in the form, allowing users to show or hide password input.

## 📸 Screenshots

Screenshot 2024-10-09 095554

### CODE REVIEW

**Comment by Sibam-Paul** [comment]
@anmol-284 Give a better background

**Comment by anmol-284** [comment]
Any one from these if you like
Screenshot 2024-10-07 213021

File: frontend/src/components/Authentication/Login.js [hunk 1]
```diff
@@ -62,8 +59,8 @@ const Login = () => {
       navigate("/chats");
     } catch (error) {
       toast({
-        title: "Error Occured!",
-        description: error.response.data?.message,
+        title: "Error Occurred!",
+        description: error.response.data.message,
```
Comment (kartikayasijaa) [inline frontend/src/components/Authentication/Login.js:63]: use `?`

**Comment by kartikayasijaa** [comment]
Just remove this bg, previous one was much better than this

**Comment by Sibam-Paul** [comment]
closes - #36

**Comment by Sibam-Paul** [comment]
@kartikayasijaa Do u like this 
image

image

**Pull request closed without merging**